### ISSUE TITLE
Provide the ability for staff to file on behalf of a corporation

### ISSUE THREAD

**Issue by trishreimer** [bcgov/business-ar#142]

**Comment by Maribeth-Wilson** [comment]
@hanlunBCRegistries - can you please take a look at this ticket and let me know if you are able to work on it? Thank you

**Comment by hanlunBCRegistries** [comment]
@Maribeth-Wilson yes I can work on it, do you mind I put a names label on it so we could estimate it in our grooming session?

**Comment by trishreimer** [comment]
Since the intent of this filing is to get people to login, create an account and affiliate, I would suggest not working on this change as a priority.

Staff can continue to file in COLIN and will have the ability to file in the Business Registry when the filing is available from the dashboard. 

@Maribeth-Wilson Why do we want to staff to file now?

**Comment by Maribeth-Wilson** [comment]
Sorry, since it was in the backlog, I thought it needed to move forward. @hanlunBCRegistries - please disregard my comment. Thank you

**Comment by ozamani9gh** [comment]
@Maribeth-Wilson do you want to move this ticket to sprint backlog on Business A/R board or should we wait for more direction from team if this is a priority at the moment.
